runtime/verbose/errormessagehelper.c

```diff
@@ -356,10 +356,10 @@ prepareVerificationTypeBuffer(StackMapFrame* stackMapFrame, MethodContextInfo* m
 		cpInfo.bytes = (U_8*)methodInfo->signature.bytes;
 		cpInfo.slot1 = (U_32)methodInfo->signature.length;
 
-		/* Calls isInitOrClinitImpl() to determine whether the method is either "<init>" or "<clinit>".
-		 * It returns 1) 0 if name is a normal name 2) 1 if '<init>' or 3) 2 if '<clinit>'
+		/* Calls isInitOrClinitOrNewImpl() to determine whether the method is "<init>", "<clinit>", or "<new>".
+		 * It returns 0 if name is a normal name, 1 if '<init>', 2 if '<clinit>', and 3 if '<new>'
```

Comment:
Please use `CFR_METHOD_NAME_INIT`, `CFR_METHOD_NAME_CLINIT` and `CFR_METHOD_NAME_NEW`, instead of 1,2,3 in this comment.